storage/backend/ceph/ceph.go

```diff
@@ -554,8 +589,13 @@ func (c *Driver) Validate(do *storage.DriverOptions) error {
 		return err
 	}
 
-	if do.Volume.Params["pool"] == "" {
-		return errored.Errorf("Pool is missing in ceph storage driver.")
+	switch dVal := do.Volume.Params["pool"].(type) {
```

Comment:
is this still required given the code above? couldn't you just check if `pool == ""`?